src/main/kotlin/uk/gov/dwp/dataworks/services/TaskDeploymentService.kt

```diff
@@ -162,6 +190,15 @@ class TaskDeploymentService {
                         "CLIENT_PARAMS" to "hostname=localhost,port=5900,disable-copy=true",
                         "CLIENT_USERNAME" to userName))
                 .portMappings(PortMapping.builder().hostPort(guacamolePort).containerPort(guacamolePort).build())
+                .logConfiguration(LogConfiguration.builder()                
+                    .logDriver("awslogs")
+                    .options(mapOf(
+                        "awslogs-group" to configurationResolver.getStringConfig(ConfigKey.CONTAINER_LOG_GROUP), 
+                        "awslogs-region" to configurationResolver.getStringConfig(ConfigKey.AWS_REGION),
+                        "awslogs-stream-prefix" to userName + "_guacamole"
+                    ))
+                    .build()
```

Comment:
I think it would be worth pulling out the `LogConfiguration` building into a dedicated function and pass in the stream name as I think that's the only thing that changes for each container?

Reply:
Done :+1: